incubator/etcd/values.yaml

```diff
@@ -21,4 +27,4 @@ Storage: "1Gi"
 ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
 ##   GKE, AWS & OpenStack)
 ##
-# storageClass: "-"
+storageClass: "-"
```

Comment:
Why is this changed? Our best practice recommends this to be commented out by default.